### DISCUSSION TITLE
Welcome to Simple-Python-Scripts Discussions!

### DISCUSSION THREAD

**Discussion by ThatSINEWAVE** [ThatSINEWAVE/Simple-Python-Scripts#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Vir-Suppae** [comment]
Are you planning on completing [contributing.md](https://github.com/ThatSINEWAVE/Simple-Python-Scripts/blob/main/CONTRIBUTING.md)?

**Reply by ThatSINEWAVE** [reply]
What is there to complete?

**Reply by Vir-Suppae** [reply]
the table of contents lists sections that are not in [contributing.md](https://github.com/ThatSINEWAVE/Simple-Python-Scripts/blob/main/CONTRIBUTING.md)

**Reply by ThatSINEWAVE** [reply]
Ah, those are not needed in this repository, there is no need for styleguides and other sections
